modules/core/src/main/java/org/apache/ignite/internal/management/snapshot/SnapshotCreateCommandArg.java

```diff
@@ -67,6 +72,17 @@ public class SnapshotCreateCommandArg extends IgniteDataTransferObject {
         dest = U.readString(in);
         sync = in.readBoolean();
         incremental = in.readBoolean();
+        onlyPrimary = in.readBoolean();
+    }
+
+    /** */
+    public boolean onlyPrimary() {
```

Comment:
There is not only the test, it also changes this class, let's move this test to a separate ticket